### DISCUSSION TITLE
Execute dbt SQL button not working

### DISCUSSION THREAD

**Discussion by G14rb** [AltimateAI/vscode-dbt-power-user#168] [Q&A] [answered]
When I execute the Execute dbt SQL command it starts looping without doing anything, is anybody else experienced the same behaviour? Thanks

**Comment by mdesmet** [comment]
Do you still have this issue in latest version. Related code has been heavily refactored in 0.8.0.

**Reply by G14rb** [reply]
I have updated to 0.8.1 and updated the dbt.queryTemplate for Sql Server but when I run 'Execute dbt SQL' I got back "Runtime Error
Database Error Attempt to use a closed connection.". But if I run dbt model normally it works.

**Reply by mdesmet** [reply]
What happens if you remove the dbt.queryTemplate config. do you get the same error?

**Reply by G14rb** [reply]
No, I get Sql Server error, "An error occured while trying to execute your query: Runtime Error Database Error ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Incorrect syntax near 'limit'. (102) (SQLExecDirectW)")",
I suppose because it tries to run 'select * from {query} limit {limit}'

**Reply by mdesmet** [reply]
Ok interesting, so that one actually has an active connection. Could you also provide me the dbt.queryTemplate that caused the other error.

**Comment by mdesmet** [comment, marked as answer]
Solved in versions above 0.8.1

**Comment by sameersk2k** [comment]
- I am facing the same issue now.
